extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonService.java

```diff
@@ -9,21 +9,69 @@
  * This service communicates with the API backend of openrefine and allows
  * direct access to rdf skeletons.
  * 
+ * The implementations of this class enable a developer to use different ways of
+ * persisting and reading rdf skeletons.
+ * 
  * @author Shamanou van Leeuwen
+ * @since 5 Oktober 2017
+ * @see http://openrefine.org/
  *
  */
 
 interface RdfSkeletonService {
 
+    /**
+     * 
+     * Returns a list of the rdf skeletons that belong to a specific filetype.
+     * 
+     * @param fileType
+     *            - the requested filetpe
+     * @return List<Model>
+     * @throws IOException
+     *             - thrown when there are no files available
+     */
     public List<Model> listModels(String fileType)
             throws IOException;
 
+    /**
+     * Returns a list of all rdf skeletons.
+     * 
+     * @return List<Model>
+     * @throws IOException
+     *             - thrown when there are no files available
```

Comment:
Is this thrown when there are no files available, or will it just return an empty list then? What happens when there is an error while reading a file?